Resources/Prototypes/_RMC14/Roles/Jobs/Marines/hospital_corpsman.yml

```diff
@@ -7,7 +7,7 @@
   requirements:
   - !type:TotalJobsTimeRequirement
     group: CMJobsMedical
-    time: 3600 # 1 hours
+    time: 7200 # 2 hours
```

Comment:
I don't think there's a reason to increase this, I'd rather have inexperienced corpsmen than not, and I'd rather the corpsmen than the nurses trying to frontline because they can't play corpsmen yet.